### ISSUE TITLE
Not a reusable app -> Repository is dead

### ISSUE THREAD

**Issue by hnykda** [jacobh/drf-fsm-transitions#12]
If I understand correctly, this app has following unjustifiable assumptions:

1. a FSMField is called `status` (e.g. in my case I call it `state`)
2. one cannot use different identifier (`pk` must be used)
3. transition must have `by` argument

Some of it seems to be PRed already.

**Comment by 27medkamal** [comment]
Agree. It seems this repository in unmaintained. I've therefore created a new package and addressed the issues above and some other issues I've encountered as well.

https://github.com/27medkamal/djangorestframework-fsm

**Comment by hnykda** [comment]
I encourage the author @jacobh to archive this repository or archive it and redirect to the above fork.

**Comment by hnykda** [comment]
@27medkamal Issues are not available at your repo though. Don't you wanna create a fresh one instead of fork?

**Comment by 27medkamal** [comment]
@hnykda thanks for letting me know. I've updated the repository
